color_number:
  blue: 51
  green: 46
  orange: 38
  purple: 43
  red: 49
  skyblue: 45
  yellow: 48
default_port: 1
number_color:
  38: orange
  43: purple
  45: skyblue
  46: green
  48: yellow
  49: red
  51: blue
number_sound:
  38: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/5029.mp3
  43: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/5022.mp3
  45: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/5013.mp3
  46: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/5017.mp3
  48: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/29927.mp3
  49: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/5012.mp3
  51: /home/daiego/catkin_ws/src/haru_drums_ros_driver/src/sounds/Minecraft_sounds/29928.mp3
